internal/assistedserviceiso/assistedserviceiso_api.go

```diff
@@ -54,10 +55,10 @@ func NewAssistedServiceISOApi(objectHandler s3wrapper.API, authHandler auth.Auth
 func (a *assistedServiceISOApi) CreateISOAndUploadToS3(ctx context.Context, params assisted_service_iso.CreateISOAndUploadToS3Params) middleware.Responder {
 	log := logutil.FromContext(ctx, a.log)
 
-	sshPublicKey := params.AssistedServiceIsoCreateParams.SSHPublicKey
-	if sshPublicKey != "" {
-		sshPublicKey = strings.TrimSpace(sshPublicKey)
-		if err := validations.ValidateSSHPublicKey(sshPublicKey); err != nil {
+	sshPublicKeys := params.AssistedServiceIsoCreateParams.SSHPublicKey
+	if sshPublicKeys != "" {
+		sshPublicKeys = strings.TrimSpace(sshPublicKeys)
```

Comment:
`TrimSpace` is already called in the `ValidateSSHPublicKey` function. Does it have to be called here too?

Reply:
You are right, it's enough to be called in `ValidateSSHPublicKey`. Thanks.